k8s/airflow-operator/manifest/application.yaml.template

```diff
@@ -37,5 +37,7 @@ spec:
     matchLabels:
       app.kubernetes.io/name: "$APP_INSTANCE_NAME"
   componentKinds:
-  - group: apps/v1beta2
-    kind: Deployment
+  - group: ""
```

Comment:
Hmm, this is correct. Which means we have been using `apiVersion` in places of `group` for a while and things still work! I just checked the code and it seems like we've been ignoring the group altogether in the deployer code.